features/step_definitions/generic_steps.rb

```diff
@@ -125,6 +125,10 @@ def assure_xpath_not_present(tag, attribute, value, selector)
   find_field("#{labeltext}").value.should == content
 end
 
+Then "the {string} input should be blank" do |label|
+  expect(find_field(label).value).to be_blank
```

Comment:
`value` can be nil for Rack::Test if [the attribute is omitted](https://web.archive.org/web/20220801180017/http://www.opinionatedprogrammer.com/2011/03/capybara-input-tag-without-value-attribute/), so I can't just do [`field should contain "^$"`](https://github.com/otwcode/otwarchive/blob/592328d48c4caaaaca9e3e87ae759cfb71a8e26b/features/step_definitions/web_steps.rb#L228-L248).